docs/apm/errors.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[errors]]
+=== Errors
+
+TIP: {apm-overview-ref-v}/errors.html[Errors] are defined as groups of exceptions with matching exception or log messages.
+
+The *errors* overview provides a high-level view of the error message and culprit,
```

Comment:
"Errors" is capitalized in the UI, so I would capitalize it here (since it's referring to the UI view)